docs/getting_started/creating_an_admin.rst

```diff
@@ -51,6 +53,9 @@ For the rest of the tutorial, you'll need some sort of model. In this tutorial,
 
     use Doctrine\Common\Collections\ArrayCollection;
 
+    /**
+     * @ORM\Entity
```

Comment:
In this case we would need the namespace and this could differ based on the persistence layer. So letzt remove the annotations but keep the link to the sf docs

Reply:
Sounds faire enought to me.